cmd/kube-proxy/app/server.go

```diff
@@ -54,7 +53,7 @@ import (
 	"k8s.io/kubernetes/pkg/proxy"
 	"k8s.io/kubernetes/pkg/proxy/apis"
 	kubeproxyconfig "k8s.io/kubernetes/pkg/proxy/apis/config"
-	"k8s.io/kubernetes/pkg/proxy/apis/config/scheme"
+	kubeproxyconfigscheme "k8s.io/kubernetes/pkg/proxy/apis/config/scheme"
```

Comment:
Wonder if it should use ``kubeproxyscheme`` instead? It is shorter.

Reply:
Indeed. but `kubeproxyconfigscheme` sync with using in `controller-manager`, and indicate the `scheme` belong to `proxy Componentconfig`. i afraid of that `kubeproxyscheme` missing the config message